sequential_parallel.py

```diff
@@ -0,0 +1,15 @@
+from threading import Thread
+
+
+def start_parallel(functions, arguments):
```

Comment:
If your working with Python 3, check out function annotations. Annotations can be very helpful to both users and developers.

Reply:
Didn't know it before, thank you
**Done!**